doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1154 @@
+      "text/plain": [
+       "                                                     error  \\\n",
+       "location_id model_name                                       \n",
+       "145         LinearRegression()                  706.173014   \n",
+       "166         LinearRegression()                  272.062997   \n",
+       "152         LinearRegression()                  266.035645   \n",
+       "145         DecisionTreeRegressor(max_depth=3)  461.833333   \n",
+       "166         DecisionTreeRegressor(max_depth=3)  227.661902   \n",
+       "\n",
+       "                                                                                       checkpoint  \\\n",
+       "location_id model_name                                                                              \n",
+       "145         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "166         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "152         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "145         DecisionTreeRegressor(max_depth=3)  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "166         DecisionTreeRegressor(max_depth=3)  /Users/christy/Documents/github_ray_temp/ray/d...   \n",
+       "\n",
+       "                                                                                   checkpoint_dir  \n",
+       "location_id model_name                                                                             \n",
+       "145         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "166         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "152         LinearRegression()                  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "145         DecisionTreeRegressor(max_depth=3)  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "166         DecisionTreeRegressor(max_depth=3)  /Users/christy/Documents/github_ray_temp/ray/d...  "
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Assemble Tune results into a pandas dataframe\n",
+    "results_df = results.get_dataframe()\n",
+    "print(results_df.columns)\n",
+    "results_df = results_df[[\"config/location\", \"config/model\", \"error\", \"logdir\"]]\n",
+    "results_df.rename(columns={'config/location':'location_id'}, inplace=True)\n",
+    "results_df.rename(columns={'config/model':'model_name'}, inplace=True)\n",
+    "results_df.set_index([\"location_id\", \"model_name\"], inplace=True)\n",
+    "results_df['checkpoint'] = results_df['logdir'].apply(lambda x: x + \"/checkpoint_000000/model\")\n",
+    "results_df['checkpoint_dir'] = results_df['logdir'].apply(lambda x: x + \"/checkpoint_000000\")\n",
```

Comment:
this is an implementation detail. instead of putting this in the dataframe, we should say that users can obtain the checkpoints from the `results` object